pkg/model/utils.go

```diff
@@ -70,6 +70,8 @@ func GetValidNameFromFolder(folder string) (string, error) {
 	name := filepath.Base(dir)
 	name = strings.ToLower(name)
 	name = ValidKubeNameRegex.ReplaceAllString(name, "-")
+	name = strings.TrimPrefix(name, "-")
+	name = strings.TrimSuffix(name, "-")
```

Comment:
@pchico83 @jLopezbarb I had to add this because in the case the folder holding the stack manifest starts or ends with a `.` (or any other invalid char for a k8s name) we were generating an invalid name like `-okteto`